src/CLI/actioner/sonic-cli-tam.py

```diff
@@ -0,0 +1,124 @@
+import sys
+import time
+import json
+import ast
+import sonic_tam_client
```

Comment:
We should not be using client sdk call because of below reason provided by mgmt-framework team:

"Introduced a new codegen.config file to control which yang models and
OpenAPI specs should be used for swagger python client sdk codegen.
Earlier client sdk was getting generated for all yangs, which takes
lot of build time and space. Also
there were performance issues since each klish CLI handler loads full
client code but uses only one function. Hence we are preferring a
generic client src/CLI/actioner/cli_client.py for CLI use. Python
client sdk will not be generated by default. Developer needs to
whitelist the yang moduels for client codegen via PY_YANGAPI_CLIENTS
variable"


Please use the alternate way , as better to use the correct way as advised by mgmt-framework team
https://github.com/project-arlo/sonic-mgmt-framework/blob/master/models/codegen.config

Reply:
Please follow the steps given in https://github.com/project-arlo/sonic-mgmt-framework/blob/master/src/CLI/actioner/README_cli_client.md